apex/contrib/test/fused_dense/test_fused_dense.py

```diff
@@ -1,6 +1,8 @@
 import unittest
-
 import torch
+import os
+from torch.testing._internal import common_utils
+from torch.testing._internal.common_device_type import instantiate_device_type_tests
```

Comment:
could you follow https://peps.python.org/pep-0008/#imports regarding the order?

is `unittest` still used?

Reply:
Will fix the order. `unittest` is still used to skip the test if for some reason apex can't import fused_dense, should I leave that there?